src/connections/VerificationUser.ts

```diff
@@ -4,23 +4,28 @@ import { verifyUser } from "@/generated/v3/userService/userServiceComponents";
 import { verifyUserFetchFailed, verifyUserIsFetching } from "@/generated/v3/userService/userServicePredicates";
 import { ApiDataStore, PendingErrorState } from "@/store/apiSlice";
 import { Connection } from "@/types/connection";
-import { connectionHook } from "@/utils/connectionShortcuts";
+import { connectionLoader } from "@/utils/connectionShortcuts";
 import { selectorCache } from "@/utils/selectorCache";
 
 export const selectVerificationUser = (store: ApiDataStore) => Object.values(store.verifications)?.[0]?.attributes;
 
 type VerificationUserConnection = {
   isLoading: boolean;
   requestFailed: PendingErrorState | null;
-  isSuccess: boolean;
-  sendVerifyUser: () => void;
+  isSuccess: boolean | null;
 };
 
 type VerificationUserProps = {
   token: string;
 };
 
 const verificationUserConnection: Connection<VerificationUserConnection, VerificationUserProps> = {
+  load: ({ isSuccess, requestFailed }, { token }) => {
+    console.log(isSuccess, requestFailed);
+    if (isSuccess === null && requestFailed === null) verifyUser({ body: { token } });
```

Comment:
Please use `==` comparison for null instead of `===`. You want to be able to catch both null and undefined here.